apps/web/src/views/GaugesVoting/components/Table/VoteTable/hooks/useGaugeRows.ts

```diff
@@ -62,7 +62,7 @@ const useSelectRowsWithQuery = (gauges: Gauge[] | undefined) => {
       const urlHashes = queryHashes.filter((hash) => !newHashes.includes(hash))
       setSelectRowsHash(urlHashes.concat(newHashes))
     }
-  }, [isLoading, prevVotedGauges, queryHashes])
+  }, [isLoading, prevVotedGauges.map((x) => x.hash).join('-'), queryHashes])
```

Comment:
will map every render, how about to use a memo value

Reply:
done